types/bluebird-global/index.d.ts

```diff
@@ -139,17 +142,17 @@ declare global {
         isRejected: Bluebird<T>["isRejected"];
         isResolved: Bluebird<T>["isResolved"];
         lastly: Bluebird<T>["lastly"];
-        map: Bluebird<T>["map"];
-        mapSeries: Bluebird<T>["mapSeries"];
+        map<U, Q>(this: Promise<T & Iterable<Q>>, mapper: IterateFunction<Q, U>, options?: Bluebird.ConcurrencyOption): Bluebird<U[]>;
+        mapSeries<U, Q>(this: Promise<T & Iterable<Q>>, iterator: IterateFunction<Q, U>): Bluebird<U[]>;
         nodeify: Bluebird<T>["nodeify"];
         props: Bluebird<T>["props"];
         race: Bluebird<T>["race"];
```

Comment:
I believe you need to change `race` as well?
The bluebird definition is:
```ts
race<Q>(this: Bluebird<R & Iterable<Q>>): Bluebird<Q>;
```

Reply:
Fixed, nice catch.